osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/services/ActionMapperService.java

```diff
@@ -239,7 +243,8 @@ public BundleMessagesRequestDto mapAllActions(final BundleMessageRequest bundleM
         return new BundleMessagesRequestDto(actionValueObjectDtoList);
     }
 
-    private ActionDto mapActionWithMapper(final SmartMeter smartMeter, final ActionRequest action) throws FunctionalException {
+    private ActionDto mapActionWithMapper(final SmartMeter smartMeter, final ActionRequest action)
+            throws FunctionalException {
         @SuppressWarnings("unchecked")
         // TODO: fix this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1135)